src/sql/migrations.rs

```diff
@@ -497,6 +497,11 @@ item TEXT DEFAULT '');"#,
         )
         .await?;
     }
+    if dbversion < 81 {
+        info!(context, "[migration] v81");
+        sql.execute_migration(r#"DELETE FROM config WHERE keyname='mvbox_watch'"#, 81)
```

Comment:
not sure, if that is worth a migration, even if that one is short and looks good, all migrations are a bit risky by definition :)

fwiw, we did not care about that when dropping config-options in the past.